samples/cellular/modem_shell/src/fota/fota.c

```diff
@@ -88,6 +88,9 @@ static void fota_download_callback(const struct fota_download_evt *evt)
 		mosh_print("FOTA: Download finished");
 		modem_update_apply();
 		break;
+	case FOTA_DOWNLOAD_EVT_ERASE_TIMEOUT:
+		mosh_print("FOTA: Erasing reached timeout");
+		/* Fall through, erasing continues. */
 	case FOTA_DOWNLOAD_EVT_ERASE_PENDING:
 		mosh_print("FOTA: Still erasing...");
 		break;
```

Comment:
@tokangas Did you check this and is this fine?

Reply:
@trantanen Yes, I did check it. As far as I understand, the timeout event gets triggered periodically, but erasing still continues. The Kconfig description for the timeout says that the LTE link could be disconnected to allow the erasing of the flash to happen more quickly, but with modem_shell I think we can leave that to the user.